app/templates/agenda/agendaitems/agendaitem/documents.hbs

```diff
@@ -18,7 +18,7 @@
           >
             {{t "batch-edit"}}
           </Auk::Button>
-        {{else}}
+        {{else if (not this.currentSession.isOverheid)}}
```

Comment:
Ik denk dat het hier zelfs groffer mag: geen button voor iedere gebruiker die geen `manage-documents` rol heeft. De read-only modal heeft geen toegevoegde waarde t.o.v. de document cards die ze al zien, hebben we vorige week met Inti bekeken.